bot/code_review_bot/revisions.py

```diff
@@ -125,7 +125,9 @@ def load_phabricator(self, try_task):
         assert self.diff_phid.startswith("PHID-DIFF-")
 
         # Load diff details to get the diff revision
-        diffs = self.api.search_diffs(diff_phid=self.diff_phid)
+        diffs = self.api.search_diffs(
+            diff_phid=self.diff_phid, attachments={"commits": True}
```

Comment:
Please add a comment mentionning that we load the commits for the build error notifications